SpeechService/SpeechService.cs

```diff
@@ -116,6 +117,16 @@ public void Say(Ship ship, string message, int priority = 3, string voice = null
                 ship = ShipDefinitions.FromModel("Sidewinder");
             }
 
+            // Adjust playback volume for VA say/transmit
+            if (volume <= 100)
+            {
+                transmitVolume = volume;
+            }
+            else
+            {
+                transmitVolume = 100;
+            }
```

Comment:
I'm not sure we need a persistent `transmitVolume` variable?

Reply:
Preventing the user from going over 100% (or perhaps 125%?) with an override volume is something that we ought to keep in any case. ;-)